README.md

```diff
@@ -40,93 +40,72 @@
-- The [packaging](https://github.com/process-analytics/bpmn-visualization-js/milestone/18) of the library.
-
-Future features:
-- BPMN extensions
-- The library extension points
-- Display options for execution data with interactive capacities.
-
 
 ## ♻️ Usage
-We support various module formats on [NPM](https://npmjs.org/package/bpmn-visualization):
-- [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)
-- [ESM](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm)
-- [CommonJS](https://www.typescriptlang.org/docs/handbook/2/modules.html#commonjs)
+Library is available through [NPM](https://npmjs.org/package/bpmn-visualization) \
+We support various module formats such as:
+[IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE),
+[ESM](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm),
+[CommonJS](https://www.typescriptlang.org/docs/handbook/2/modules.html#commonjs).
   
-
-### IIFE
-See how it's done in [this example](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/01-getting-started).
-
-
-### ESM
-🚀 You can find an integration example of Vanilla Typescript with rollup, [here](https://github.com/process-analytics/bpmn-visualization-examples/tree/master/examples/projects/typescript-vanilla-with-rollup).
+Here are examples per module: \
+[bpmn-visualization (IIFE)](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/01-getting-started) \
+[bpmn-visualization (ESM); Typescript + Rollup](https://github.com/process-analytics/bpmn-visualization-examples/tree/master/examples/projects/typescript-vanilla-with-rollup)
 
 ⚒️ If you want to configure yourself:
 * Install the dependency in your **package.json** file:
 ```shell script
 npm i bpmn-visualization
 ```
 
-* Load necessary scripts 
+* Load bpmn-visualization script (replace _X._Y._Z by the recent version)
 ```html
-    <!-- load global settings -->
-    <script src="./static/js/configureMxGraphGlobals.js"></script>
-    <!-- load mxGraph client library -->
-    <script src="./static/js/mxClient.min.js"></script>
-    <!-- load BPMN Visualization library -->
-    <script src="index.esm.js" type="module"></script>
+    <script src="https://unpkg.com/bpmn-visualization@_X._Y._Z/dist/bpmn-visualization.js"></script>
 ```
 * Define your BPMN content using one of the following ways:
   * Copy/Paste directly the XML content in a variable
   * Load it from a url, like this [example](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/load-remote-bpmn-diagrams/index.html)
   * Load from your computer, like the [demo example](https://github.com/process-analytics/bpmn-visualization-js/blob/master/src/demo/index.ts)
 ```javascript
-    const bpmnContent = ``; // your BPMN 2.0 XML content
+    function bpmnDiagram() {
+      return  ``; // your BPMN 2.0 XML content
+    }
 ```
-* Define the mxGraph container
+* Define the bpmn diagram container
```

Comment:
```suggestion
* In the HTML page, define the container that displays the BPMN diagram
```

Reply:
I don't like switching from js to html then js back. Not fluid and messy

Reply:
Me too :)